examples/typescript/evm/importAccount.ts

```diff
@@ -0,0 +1,11 @@
+// Usage: pnpm tsx evm/importAccount.ts
```

Comment:
Can you run the formatter on this file? Easiest way to do that now is from the root `typescript/` folder, run: `pnpm prettier -c .prettierrc --write ../examples/typescript/evm/importAccount.ts`

Reply:
done!